orchestrator/relayer/src/batch_relaying.rs

```diff
@@ -74,14 +75,28 @@ async fn get_batches_and_signatures(
     current_valset: Valset,
     grpc_client: &mut GravityQueryClient<Channel>,
     gravity_id: String,
+    supported_contracts: Vec<EthAddress>,
 ) -> HashMap<EthAddress, Vec<SubmittableBatch>> {
-    let latest_batches = if let Ok(lb) = get_latest_transaction_batches(grpc_client).await {
-        lb
+    let mut latest_batches: Vec<TransactionBatch> = Vec::new();
+    if supported_contracts.len() == 0 {
```

Comment:
nit: could be `supported_contracts.is_empty()`?

Reply:
done